CLUSTER.md

```diff
@@ -0,0 +1,31 @@
+# GTSFM on CLUSTER
+
+### How to Run GTSfM on a Cluster?
+
+GTSfM uses [SSHCluster](https://docs.dask.org/en/stable/deploying-ssh.html#dask.distributed.SSHCluster) module of [Dask.distributed](https://distributed.dask.org/en/stable/) library to allow utilizing clusters for GTSfM runs. This is a step-by-step guide on how to set up your machines for a successful run on a cluster.
+
+1. Enable passwordless SSH between all the workers on the cluster
```

Comment:
nit: should step 1 be to choose which machine will serve as the "scheduler node"?

Reply:
Done.

Reply:
Resolved, thanks!